### DISCUSSION TITLE
Restore EE2 ability to force a 'master sort' on Category Groups to be Alphabetical

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#3150] [Feature Requests]
EE2 used to let us force a particular Category Group to always be Alphabetical - meaning the Drag & Drop ordering in the CP is disabled and adding new categories automatically sorts them Alphabetical.  Here's the interface in V2:
<img width="153" alt="image" src="https://user-images.githubusercontent.com/8193519/226643174-7273f13c-a213-4113-8583-8726b599506f.png">

I'm not sure when this disappeared, but this sort of control per category group could be useful in some situations.

**Comment by diarbyrag** [comment]
I'm all for it as I have a project I'm working on that has 5/6 category groups with around 12 parent categories in each and some of them have up to 30 child categories, each group is basically a different language version of the same parent/child categories. If ee were allowed to sort them alphabetically it would be crucial however that it sorted parents and children independently so they do not get jumbled up.
It would however be a big time saver, expecially when I'm dealing with non english categories (which although they are all the same category parent/child types) they are obviously spelled differently and need sorting accordingly.
... hope that makes sense.
the very first site I ever did was in ee2 and it was great when ee did the initial sorting, although you could override it where necessary
